adal/tests/Test.ADAL.NET.UIAutomation/AppInitializer.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using Xamarin.UITest;
+using Xamarin.UITest.Queries;
+
+namespace Test.ADAL.NET.UIAutomation
+{
+	public class AppInitializer
+	{
+		public static IApp StartApp(Platform platform)
+		{
+			if (platform == Platform.Android)
+			{
+				return ConfigureApp.Android.InstalledApp("com.Microsoft.XFormsDroid").StartApp();
+
+            }
+
+			return ConfigureApp.iOS.StartApp();
```

Comment:
It'll be safer to add a switch statement over that Platform enum and to return an app for each supported platform and a NotImplemented or NotSupported exception for the other ones.